cuml/kmeans_demo.ipynb

```diff
@@ -146,15 +146,15 @@
    "outputs": [],
    "source": [
     "fig = plt.figure(figsize=(16, 10))\n",
-    "plt.scatter(host_data[:, 0], host_data[:, 1], c=host_labels, s=50, cmap='viridis')\n",
+    "plt.scatter(host_data.iloc[:, 0], host_data.iloc[:, 1], c=host_labels, s=50, cmap='viridis')\n",
     "\n",
     "#plot the sklearn kmeans centers with blue filled circles\n",
     "centers_sk = kmeans_sk.cluster_centers_\n",
     "plt.scatter(centers_sk[:,0], centers_sk[:,1], c='blue', s=100, alpha=.5)\n",
     "\n",
     "#plot the cuml kmeans centers with red circle outlines\n",
     "centers_cuml = kmeans_cuml.cluster_centers_\n",
-    "plt.scatter(centers_cuml['0'], centers_cuml['1'], facecolors = 'none', edgecolors='red', s=100)\n",
+    "plt.scatter(centers_cuml['0'].values, centers_cuml['1'].values, facecolors = 'none', edgecolors='red', s=100)\n",
```

Comment:
I'm running your changes and getting this error:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-7-4a89ad5b0ed4> in <module>
      8 #plot the cuml kmeans centers with red circle outlines
      9 centers_cuml = kmeans_cuml.cluster_centers_
---> 10 plt.scatter(centers_cuml['0'].values, centers_cuml['1'].values, facecolors = 'none', edgecolors='red', s=100)
     11 
     12 plt.title('cuml and sklearn kmeans clustering')

AttributeError: 'Series' object has no attribute 'values'
```

It might help to make sure we're on the same versions. 

I'm running `cudf 0.9.0a py36_1267` and Pandas `0.24.2 py36hb3f55d8_0`

Reply:
The error does disappear for me if I take away the `.values`

Reply:
the cudf i have is :
```
cudf                      0.9.0a                py37_2773    rapidsai-nightly
pandas                    0.24.2           py37hb3f55d8_0    conda-forge
```